code_schemes/kakuma_ws_correct_dataset.json

```diff
@@ -0,0 +1,584 @@
+  "Version": "0.0.0.1",
+  "Codes": [
+    {
+      "CodeID": "code-ac8223be",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma household language",
+      "StringValue": "kakuma_household_language",
+      "NumericValue": 1,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma household language"
+      ]
+    },
+    {
+      "CodeID": "code-5d39be18",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma age",
+      "StringValue": "kakuma_age",
+      "NumericValue": 2,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma age"
+      ]
+    },
+    {
+      "CodeID": "code-7e3694d6",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma gender",
+      "StringValue": "kakuma_gender",
+      "NumericValue": 3,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma gender"
+      ]
+    },
+    {
+      "CodeID": "code-2afd82b5",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma location",
+      "StringValue": "kakuma_location",
+      "NumericValue": 4,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma location"
+      ]
+    },
+    {
+      "CodeID": "code-a9a51f4a",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma nationality",
+      "StringValue": "kakuma_nationality",
+      "NumericValue": 5,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma nationality"
+      ]
+    },
+    {
+      "CodeID": "code-18a80f1b",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma leap s01e01",
+      "StringValue": "kakuma_leap_s01e01",
+      "NumericValue": 6,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma leap s01e01"
+      ]
+    },
+    {
+      "CodeID": "code-b7e78e4e",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma leap s01e02",
+      "StringValue": "kakuma_leap_s01e02",
+      "NumericValue": 7,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma leap s01e02"
+      ]
+    },
+    {
+      "CodeID": "code-18ef9e65",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma leap s01e03",
+      "StringValue": "kakuma_leap_s01e03",
+      "NumericValue": 8,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma leap s01e03"
+      ]
+    },
+    {
+      "CodeID": "code-1f005d00",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma leap s01e04",
+      "StringValue": "kakuma_leap_s01e04",
+      "NumericValue": 9,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma leap s01e04"
+      ]
+    },
+    {
+      "CodeID": "code-3611e0c8",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma leap s01e05",
+      "StringValue": "kakuma_leap_s01e05",
+      "NumericValue": 10,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "kakuma leap s01e05"
+      ]
+    },
+    {
+      "CodeID": "code-ba352823",
+      "CodeType": "Normal",
+      "DisplayText": "kakuma s02e01",
```

Comment:
I wonder if it's worth tweaking the display text to e.g. kakuma keep ii s02e01, just to make the difference super clear.

Reply:
A good suggestion to reduce coding errors. Updated for DisplayText only for compatibility with older datasets.